core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -195,6 +196,7 @@
   @Before
   public void before() throws Exception {
     GroupMappingServiceTestUtils.resetCache();
+    MetricsSystem.clearAllMetrics();
```

Comment:
Could this be called in LocalAlluxioCluster.stop() for a wider impact?

Reply:
Integration tests already disable metrics, so they don't have the issue
https://github.com/Alluxio/alluxio/blob/master/minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java#L315

FileSystemMasterTest sets masters up directly instead of using LocalAlluxioCluster

Reply:
Are there master-side metrics that could be getting accumulated regardless of this setting?
I just tried resetting the MetricSystem at LocalAlluxioCluster resource and test runs benefited immediately. FileSystemMasterIntegrationTest run time came down consistenly to ~42 secs from ~60.

Reply:
oh, I think it must still be aggregating worker metrics. I tried clearing metrics before every test but hit some test failures. Lets merge this for now, then deal with integration tests in a separate PR